### PR TITLE
Update node modules and fix casing of some components

### PR DESCRIPTION
**Pull request by pvenable**
This includes React 17.0.1 and a storybook update that required some small changes.

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -20,5 +21,6 @@ addDecorator((story) => (
 ));
 addDecorator((story) => <ThemeProvider theme={theme}>{story()}</ThemeProvider>);
 addDecorator(withA11y);
+addDecorator(withKnobs);
```
Comment (pvenable) [inline .storybook/config.js:24]: Moved this to a global decorator as storybook started throwing errors with the "local" ones (or at least the one in the Accordion stories).

File: jest.setup.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```
Comment (pvenable) [inline jest.setup.js:2]: The official enzyme adapter for react 17 isn't out yet, but this fork works for now.

File: README.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ import { ThemeProvider } from 'emotion-theming';
 function App() {
   return (
     <ThemeProvider theme={theme}>
-      <Heading.h1>Hello World</Heading.h1>
+      <Heading.H1>Hello World</Heading.H1>
```
Comment (pvenable) [inline README.md:39]: Updated in accordance with the casing changes herein.

File: src/Heading/Heading.js [hunk 4]
```diff
@@ -36,38 +36,38 @@ const createHeading = (h) => (
   </Heading>
 );
 
-Heading.h1 = createHeading('h1');
-Heading.h1.defaultProps = {
+Heading.H1 = createHeading('h1');
+Heading.H1.defaultProps = {
```
Comment (pvenable) [inline src/Heading/Heading.js:40]: Each of these headings changed to adhere to PascalCasing.

File: src/Text/Text.js [hunk 5]
```diff
@@ -60,6 +60,6 @@ Text.defaultProps = {
 };
 
 // Deprecated -- use <Text as="span"></Text>
-Text.span = (props) => <Text as="span" {...props} />;
+Text.Span = (props) => <Text as="span" {...props} />;
```
Comment (pvenable) [inline src/Text/Text.js:63]: Also changed to PascalCase.

**Review by xxx — APPROVED** [review]

**Pull request merged**